bot/responses/not_sending_videos/transcription_handler_responses.py

```diff
@@ -6,14 +6,59 @@
 
 
 def get_no_quote_provided_message() -> str:
-    return "🔎 Podaj cytat, który chcesz znaleźć. Przykład: /klip Nie szkoda panu tego pięknego gabinetu?"
+    return "🔎 Podaj cytat, który chcesz znaleźć. Przykład: /transkrypcja Nie szkoda panu tego pięknego gabinetu?"
 
 
-def get_transcription_response(quote: str, context_segments: List[Dict[str, Union[int, str]]]) -> str:
-    response = f"🔍 Transkrypcja dla cytatu: \"{quote}\"\n\n```\n"
-    for segment in context_segments:
-        response += f"🆔 {segment["id"]} - {segment["text"]}\n"
+def get_transcription_response(
+    quote: str, result: Dict[
+        str, Union[
+            float,
+            Dict[str, Union[int, Dict[str, Union[int, str]], str]], List[Dict[str, Union[int, str]]],
+        ],
+    ],
+) -> str:
+    start_time = float(result["overall_start_time"])
+    end_time = float(result["overall_end_time"])
+
+    episode_info = result.get("target", {}).get("episode_info", {})
+
+    season = episode_info.get("season")
+    episode_number = episode_info.get("episode_number")
+    episode_title = episode_info.get("title")
+
+    if any(
+            not isinstance(value, expected_type) for value, expected_type in
+            ((season, int), (episode_number, int), (episode_title, str))
+    ):
```

Comment:
jesli to jest realizacja mojego pomyslu z any to go nie realizuj XDDDDD wyszlo krzywo strasznie